test/File/RenameUploadTest.php

```diff
@@ -412,4 +412,30 @@ public function testReturnUnfiltered($input)
 
         $this->assertEquals($input, $filter($input));
     }
+
+    /**
+     * @see https://github.com/zendframework/zend-filter/issues/77
+     */
+    public function testBackwordCompatibilityBreakFromRelease280ToRelease290()
```

Comment:
This test runs successfully on [release-2.8.0](https://github.com/zendframework/zend-filter/releases/tag/release-2.8.0)

Reply:
Let's give it a name that details the _behavior_ being tested, not the _symptom_. :smile:

Something like `testCachesResultsOfFilteringSAPIUploads()`.

(I can make that change on merge.)